src/docs/get-started/flutter-for/ios-devs.md

```diff
@@ -200,13 +206,13 @@ Widget build(BuildContext context) {
     ),
   );
 }
-{% endprettify %}
+```
 
-You can add padding to any widget, which mimics the functionality of
-constraints in iOS.
+You can add padding to any widget,
+which mimics the functionality of constraints in iOS.
 
-You can view the layouts that Flutter has to offer in the [widget
-catalog](/docs/development/ui/widgets/layout).
+You can view the layouts that Flutter has to offer in the
```

Comment:
This wording left me with the impression that Flutter only has a limited number of layouts to offer.